command/command.go

```diff
@@ -148,6 +148,47 @@ func loadYAMLFiles(fsys fs.FS, paths []string) (loadResults, error) {
 	return r, nil
 }
 
+// FixIndentation corrects the indentation for the given loadResult and edits it in-place.
```

Comment:
It might be better to call this in `loadYAMLFiles` natively instead of requiring an explicit call? In there, we already have the raw contents too, so it's less reading/writing.

Reply:
IMO: Adding this to loadYAMLFiles feels like a side-effect. It means that if someone wants to load a yaml file and get the unaltered content it isn't possible without writing yet another method. Additionally, this is unnecessary work for a method like `check` but we'd force it to do the work to format indentation even though it doesn't really make sense.

If you still feel this is a good idea I would recommend renaming the method to something like loadAndFormatYAMLFiles

Reply:
we could change the signature: `loadYAMLFiles(src string, format bool)`?

Reply:
Done